Lib/fontTools/pens/hashPointPen.py

```diff
@@ -33,8 +53,11 @@ class HashPointPen(AbstractPointPen):
     >    pass
     """
 
-    def __init__(self, glyphWidth=0, glyphSet=None):
+    def __init__(
+        self, glyphWidth=0, glyphSet=None, roundFunction: Callable | None = None
```

Comment:
instead of optional callable, you could set the default to `fontTools.misc.roundTools.noRound`

Reply:
I figured it would be better to check for None once, instead of rounding 4 values with a no-op function :)

Reply:
have you tested it? I think there will likely be a negligible difference between the approaches, since modern Python interpreters are highly optimized. We use the `noRound` approach in other parts of the code like varLib.varStore which is arguably more perf critical than this pen. In general I think it's more elegant to use a no-op function.